eZ/Publish/Core/Search/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -223,6 +223,7 @@ protected function getTotalCount( Criterion $criterion, $sortClauses, array $fie
      */
     protected function getFieldMap( $sortClauses )
     {
+        return array();
```

Comment:
ups, what's this? it is returning without doing anything? if so, code below could be deleted?